src/siteconfig/models.py

```diff
@@ -6,13 +6,41 @@
 from django.dispatch import receiver
 from django.shortcuts import get_object_or_404
 from django.templatetags.static import static
+from django.conf import settings
 
 from django_tenants.utils import get_public_schema_name
 from redis import exceptions as redis_exceptions
 
 User = get_user_model()
 
 
+def get_default_deck_owner():
+    """ 
+        This is run once during site initialization
+
+        Also need to initialize owner if none exists yet.
+        (Since initialization.py loads after everything else)
+    """ 
+    # return default owner if it exists. This will run after initialization.py runs
+    owner_user_qs = User.objects.filter(username=settings.TENANT_DEFAULT_OWNER_USERNAME, is_superuser=True, is_staff=True)
+    if owner_user_qs.exists():
+        return owner_user_qs.first().pk
```

Comment:
Lack of consistency: returning pk here but return id down below in same method.  They are the same think in this case (by default pk = id, but pk can be set to something else).  There's probably a proper one (pk or id) that foreign keys are supposed to use, can you look it up and change them to the correct one?

Reply:
seems like pk is preferable https://stackoverflow.com/a/2165880

Reply:
done